packages/synapse-react-client/src/lib/containers/AccessRequirementListV2/RequirementItem/SelfSignAccessRequirementItem.tsx

```diff
@@ -0,0 +1,161 @@
+ * {@link TermsOfUseAccessRequirement}.
+ *
+ * For {@link ManagedACTAccessRequirement}s, see {@link RequestDataAccess}
+ */
+export default function SelfSignAccessRequirementItem(
+  props: SelfSignAccessRequirementItemProps,
+) {
+  const { accessRequirement, onHide } = props
+  const { accessToken } = useSynapseContext()
+  const isSignedIn = !!accessToken
+  const isTermsOfUse = isTermsOfUseAccessRequirement(accessRequirement)
+
+  const { data: user } = useGetCurrentUserProfile()
+  const { data: userBundle } = useGetCurrentUserBundle()
+
+  const certificationRequirementNotMet =
+    isSignedIn &&
+    userBundle &&
+    !isTermsOfUse &&
+    accessRequirement.isCertifiedUserRequired &&
+    !userBundle.isCertified
+  const verificationRequirementNotMet =
+    isSignedIn &&
+    userBundle &&
+    !isTermsOfUse &&
+    accessRequirement.isValidatedProfileRequired &&
+    !userBundle.isVerified
+
+  const { data: accessRequirementStatus, isLoading: isLoadingStatus } =
+    useGetAccessRequirementStatus(String(accessRequirement.id))
+
+  const isApproved = accessRequirementStatus?.isApproved
+
+  const { data: wikiPage } = useGetAccessRequirementWikiPageKey(
+    accessRequirement.id.toString(),
+    // TermsOfUse ARs have the terms embedded in the AR, there is no Wiki.
+    { enabled: !isTermsOfUse },
+  )
+
+  const [showTerms, setShowTerms] = useState<boolean>(false)
+
+  const { mutate: createAccessApproval, isLoading } = useCreateAccessApproval({
+    onError: err => {
+      console.error('Error creating access approval: ', err)
+    },
+  })
+
+  const onAcceptClicked = () => {
+    if (!isApproved) {
+      const accessApprovalRequest: AccessApproval = {
+        requirementId: accessRequirement?.id,
+        submitterId: user?.ownerId!,
+        accessorId: user?.ownerId!,
+        state: ApprovalState.APPROVED,
+      }
+
+      createAccessApproval(accessApprovalRequest)
+    }
+  }
+
+  let renderedTerms = <></>
+  if (isTermsOfUse) {
+    renderedTerms = <MarkdownSynapse markdown={accessRequirement.termsOfUse} />
+  } else if (wikiPage) {
+    renderedTerms = (
+      <MarkdownSynapse
+        wikiId={wikiPage?.wikiPageId}
+        ownerId={wikiPage?.ownerObjectId}
+        objectType={wikiPage?.ownerObjectType}
+      />
+    )
+  }
+  let actions: ButtonProps[] | undefined = undefined
+
+  if (!!accessToken && !isApproved) {
+    let acceptButtonText = 'I Accept Terms of Use'
+    let acceptButtonDisabled = false
+    if (certificationRequirementNotMet || verificationRequirementNotMet) {
+      acceptButtonDisabled = true
+      acceptButtonText = `You must ${
+        certificationRequirementNotMet ? 'be certified' : ''
+      }${
+        certificationRequirementNotMet && verificationRequirementNotMet
+          ? ' and '
+          : ''
+      }${
+        verificationRequirementNotMet ? 'have a validated user profile' : ''
+      } to accept`
+    }
```

Comment:
This chunk fixes PORTALS-2644. Disable the button and update the text to tell the user that they need to complete certification/validation before they can accept.

Reply:
![image](https://user-images.githubusercontent.com/17580037/234883023-de0cf08a-eefe-4d64-b846-320a62dc38a4.png)

Reply:
Refer to design to improve